pkg/controllers/securitypolicy/securitypolicy_controller.go

```diff
@@ -23,35 +23,32 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/event"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
 	"github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1"
+	"github.com/vmware-tanzu/nsx-operator/pkg/controllers/common"
 	"github.com/vmware-tanzu/nsx-operator/pkg/metrics"
 	_ "github.com/vmware-tanzu/nsx-operator/pkg/nsx/ratelimiter"
-	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/securitypolicy"
 	util2 "github.com/vmware-tanzu/nsx-operator/pkg/nsx/util"
 	"github.com/vmware-tanzu/nsx-operator/pkg/util"
 )
 
-const (
-	METRIC_RES_TYPE = "securitypolicy"
-)
-
 var (
-	log                     = logf.Log.WithName("controller").WithName("securitypolicy")
-	resultNormal            = ctrl.Result{}
-	resultRequeue           = ctrl.Result{Requeue: true}
-	resultRequeueAfter5mins = ctrl.Result{Requeue: true, RequeueAfter: 5 * time.Minute}
+	log                     = common.Log
```

Comment:
Your code is changing the `log` from `logf.Log.WithName("controller").WithName("securitypolicy")` to `logf.Log.WithName("controller")`, I don't think it's a good idea, because we cannot ensure that every developer can append the controller name in the parameters of `log`.

Reply:
As discussed, since there is no difference in log output using WithName, we could use a global logger.